tests/link/jax/test_scan.py

```diff
@@ -17,7 +19,8 @@
 from tests.link.jax.test_basic import compare_jax_and_py
 
 
-jax = pytest.importorskip("jax")
+jax.config.update("jax_platform_name", "cpu")
+config.floatX = "float32"
```

Comment:
We shouldn't force it to be "float32". I just meant you can use this to debug locally, when you see the CI that runs on float32 failing

Reply:
I knew that, just forgot to remote it before I committed. Very cringe.